<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::TaskBatchRunnerStatusView</class>
 <widget class="QWidget" name="Elpida::TaskBatchRunnerStatusView">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="bgRunDetails">
     <property name="title">
      <string>Run Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblStatus">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblSatusValue">
        <property name="text">
         <string>&lt;span style=&quot;color:#008d09;&quot;&gt;Ready&lt;/span&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblCurrentTaskBatch">
        <property name="text">
         <string>Current Task Batch:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblCurrentTaskBatchName">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblCurrentTask">
        <property name="text">
         <string>Current Task:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblCurrentTaskName">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QProgressBar" name="pbRunnerProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
